packages/suite/src/views/wallet/coinmarket/common/CoinmarketForm/CoinmarketFormInputs.tsx

```diff
@@ -224,9 +225,10 @@ export const CoinmarketFormInputs = () => {
         );
     }
 
-    const { buyInfo } = context;
+    const { buyInfo, device } = context;
     const { currencySelect, cryptoSelect } = context.getValues();
     const supportedCryptoCurrencies = buyInfo?.supportedCryptoCurrencies;
+    const isBitcoinOnly = device?.firmwareType === FirmwareType.BitcoinOnly;
```

Comment:
please use util `hasBitcoinOnlyFirmware`

Reply:
please refactor it on all places in trade section (in case it is also somewhere else in suite 🥺 🙏  you can too)

Reply:
look for `device?.firmwareType === FirmwareType.BitcoinOnly` and `device.firmwareType === FirmwareType.BitcoinOnly`